distsql/select_result.go

```diff
@@ -248,16 +247,12 @@ func (r *selectResult) updateCopRuntimeStats(ctx context.Context, detail *execde
 		return
 	}
 
-	r.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl.RecordOneReaderStats(r.rootPlanID.String(), respTime, detail)
+	r.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl.RecordOneReaderStats(r.rootPlanID, respTime, detail)
 	for i, detail := range r.selectResp.GetExecutionSummaries() {
 		if detail != nil && detail.TimeProcessedNs != nil &&
 			detail.NumProducedRows != nil && detail.NumIterations != nil {
-			planID := ""
-			if detail.GetExecutorId() != "" {
```

Comment:
Why delete this if branch?

Reply:
Currently, `detail.GetExecutorId()` returns `string`, but I need `int` as id now. So I add a `Fixme` here to fix this proto buffer later.